compiler/cpp/src/generate/t_java_generator.cc

```diff
@@ -441,7 +441,7 @@ string t_java_generator::java_type_imports() {
 }
 
 string t_java_generator::java_suppressions() {
-  return "@SuppressWarnings({\"cast\", \"rawtypes\", \"serial\", \"unchecked\"})\n";
+  return "@SuppressWarnings(\"all\")\n";
```

Comment:
Ok, I've changed the commit to just add "unused" warnings to the suppressions list.